build/release/Makefile

```diff
@@ -156,10 +156,19 @@ define chart.targets
 promote.chart.$(1).$(2):
 	@echo === helm push $(2) to $(1)
 	$(HELM) push $(HELM_OUTPUT_DIR)/$(2)-$(HELM_CHART_VERSION).tgz oci://$(1)
-promote.all.charts: promote.chart.$(1).$(2)
+promote.all.charts: promote.chart.$(1).$(2) sign.chart.$(1).$(2)
```

Comment:
With makefile dependencies, ordering is very important since we need to sign before publishing. Do we need to swich the order of the `sign.chart` and `promote.chart`?

Reply:
Does it look good, not exactly sure of helm release process